Formula/grafana.rb

```diff
@@ -0,0 +1,158 @@
+require "language/node"
+
+class Grafana < Formula
+  desc "Gorgeous metric visualizations and dashboards for timeseries databases."
+  homepage "http://grafana.org"
+  url "https://github.com/grafana/grafana/archive/v3.1.0.tar.gz"
+  sha256 "658c858e905ecb05e03aabf57ce02184b484f98e1fb979b67d26e16dfd93196a"
+
+  head "https://github.com/grafana/grafana.git"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    grafana_path = buildpath/"src/github.com/grafana/grafana"
+    grafana_path.install Dir["*"]
+    grafana_path.install ".jscs.json", ".jsfmtrc", ".jshintrc", ".bowerrc"
+
+    cd grafana_path do
+      system "go", "run", "build.go", "setup"
+      system "go", "run", "build.go", "build"
+      system "npm", "install", *Language::Node.local_npm_install_args
+      system "npm", "install", "grunt-cli", *Language::Node.local_npm_install_args
+      system "node_modules/grunt-cli/bin/grunt", "build"
+
+      bin.install "bin/grafana-cli"
+      bin.install "bin/grafana-server"
+      (bin/"grafana").write(env_script)
+      chmod 0755, bin/"grafana"
```

Comment:
This line shouldn't be necessary since permissions are set/"corrected" automatically by brew during installation.